pymongo/cursor.py

```diff
@@ -590,6 +590,7 @@ def __getitem__(self, index):
             clone = self.clone()
             clone.skip(index + self.__skip)
             clone.limit(-1)  # use a hard limit
+            clone.__query_flags &= ~CursorType.TAILABLE  # PYTHON-1371
```

Comment:
I haven't tested it, but I assume awaitData has the same problem. This should probably use CursorType.TAILABLE_AWAIT.

Reply:
@behackett You are right. I miss the 'awaitData' bit.